google/auth/credentials.py

```diff
@@ -98,9 +101,7 @@ def valid(self):
 
     @property
     def token_state(self):
-        """
-        See `:obj:`TokenState`
-        """
+        """See `:obj:`TokenState`"""
```

Comment:
I'm curious why so many formatting changes happened.

Did you use `nox -s blacken`?